### DISCUSSION TITLE
We take long time and share our experience free of cost

### DISCUSSION THREAD

**Discussion by IrfanGhuori** [community/community#118801] [New to GitHub]
People use it and some copy it. But no one returns feedback or stars with no motivation on GitHub. we don't know how many people downloaded it and how many cloned projects. I am looking for a better website to upload all projects.

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @IrfanGhuori!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Feed category is solely related to conversations around the GitHub product Feed. This question should be in the General category. I've gone ahead and moved it for you. Good luck!
